app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -192,22 +193,63 @@ class SamplesView extends React.Component {
         ? mapSidebarSelectedSampleIds
         : selectedSampleIds;
 
-    const log = () =>
-      logAnalyticsEvent("SamplesView_heatmap-icon_clicked", {
-        selectedSampleIds: targetSampleIds.length,
+    const heatmapOptions = [
```

Comment:
Nit: You can move heatmapOptions inside `if (this.props.admin)`. No need to `push` on a separate line.